app/src/main/java/org/greenstand/android/TreeTracker/models/location/LocationDataCapturer.kt

```diff
@@ -75,12 +75,12 @@ class LocationDataCapturer(
                     val longStdDev = currentConvergence?.longitudinalStandardDeviation()
                     val latStdDev = currentConvergence?.latitudinalStandardDeviation()
                     if (longStdDev != null && latStdDev != null) {
-//                        val minimumConvergenceRatio = min(locationDataConfig.latStdDevThreshold.div(latStdDev).toFloat(),locationDataConfig.lonStdDevThreshold.div(longStdDev).toFloat())
-//                        percentageConvergence = min(1f,minimumConvergenceRatio)
-//                        Timber.d(
-//                            "Percentage convergence " +
-//                                    "[${percentageConvergence}]. \n "
-//                        )
+                        val minimumConvergenceRatio = min(locationDataConfig.latStdDevThreshold.div(latStdDev).toFloat(),locationDataConfig.lonStdDevThreshold.div(longStdDev).toFloat())
+                        percentageConvergence = min(1f,minimumConvergenceRatio)
+                        Timber.d(
```

Comment:
@Elforama this is where I logged the value which was been updated as the capture converges